docs/c-runtime-library/routine-mappings.md

```diff
@@ -1,7 +1,7 @@
 ---
-description: "Learn more about: Microsoft specific generic-text functions and the CRT functions they map to."
 title: "Generic-text function mappings"
-ms.date: "11/04/2016"
+description: "Learn more about: Microsoft specific generic-text functions and the CRT functions they map to."
+ms.date: 11/04/2016
 ms.author: twhitney
 api_name: ["foo",]
```

Comment:
@TylerMSFT Off-topic for this PR, but is the addition of this line intentional in commit https://github.com/MicrosoftDocs/cpp-docs/commit/e27b0e0952bfc46c54fc509e9e4e37b2b1389712.